deploy/install.sh

```diff
@@ -50,6 +56,21 @@ if [ -z "$PAAS_TOKEN" ]; then
   exit 1
 fi
 
+if [ -z "$CLUSTER_NAME" ]; then
+  echo "Error: cluster name not set!"
+  exit 1
+fi
+
+if ! echo "$CLUSTER_NAME" | grep -q "$CLUSTER_NAME_REGEX"; then
```

Comment:
`grep` doesn't match regex by default, you want to use `-E` in Linux. No idea about Mac.

Bash also supports regexes, but you'd need to update the shebang.

Reply:
Added -E